src/blocks/long_description.js

```diff
@@ -9,7 +9,7 @@ const LongDescription = ( page ) => {
 
 	return (
 		<div className="long-description-accordion">
-			<Accordion header={ page.title }>
+			<Accordion header="Full text description of image">
```

Comment:
Should we make this more dynamic incase it is used to describe something that isn't an image? It could be a `More information` slider in the future. Happy for it to stay as is just worth talking about.

Reply:
This was a requirement for the module.